app/locker/repository/Client/EloquentRepository.php

```diff
@@ -63,7 +64,7 @@ protected function constructStore(Model $model, array $data, array $opts) {
     ];
     $model->lrs_id = $opts['lrs_id'];
     $model->authority = $data['authority'];
-    $model->scopes = ['all'];
+    $model->scopes = $data['scopes'];
```

Comment:
Ah this needs to default to ['all'] or the [ClientController@create](https://github.com/LearningLocker/learninglocker/blob/develop/app/controllers/ClientController.php#L77) needs to change to supply the scopes. The ClientController provides another API for creating clients although I believe it can't be used outside of the UI because of the auth required.